README.md

```diff
@@ -1,72 +1,46 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TO-DO-LIST
 
-> Description the project.
+> The purpose of this project is to create a web page application that has a to-do list,when the user adds items the list is generated and displayed on the page. We use webpack javascript and basic html and css.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- HTML, CSS, JAVASCRIPT & WEBPACK
 
+## ScreenShot Demo
+![AddBook Page](./images/page.png)
 
-To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+## Live Demo (if available)
 
+[https://lyangend.github.io/TO-DO-LIST/]
 
 
-## Authors
+## Getting Started
+To get a local copy and run:
+git@github.com:LYANGEND/TO-DO-LIST.git in your terminal.
```

Comment:
Can  we use block code syntax here? This helps the README more readable. you can [check out  on how to write block syntax](https://www.markdownguide.org/basic-syntax/#code-blocks-1) for more details